packages/web-components/src/components/rux-tabs/rux-tabs.tsx

```diff
@@ -129,8 +130,23 @@ export class RuxTabs {
         this._addTabs()
     }
 
+    componentWillUpdate() {
+        this._checkSelected()
+    }
+
     private _addTabs() {
-        this._tabs = Array.from(this.el.querySelectorAll('rux-tab'))
+        this._tabs = Array.from(this.el?.querySelectorAll('rux-tab'))
+    }
+
+    private _checkSelected() {
+        // If no selected tab exists, we need to set an item to tabindex = 0
```

Comment:
So the keyboard functionality we implemented is based on this documentation from w3
[https://www.w3.org/WAI/ARIA/apg/patterns/tabs/](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)

We could change the way ours works so that you can just tab through them all.. (possibly breaking? don't know) 
Looks like shoelace allows both:
https://shoelace.style/components/tab-group

Carbon works with arrow keys with tab taking you to the next element:
https://carbondesignsystem.com/components/tabs/usage/

Calcite also uses arrows and tab takes you to the next element:
https://developers.arcgis.com/calcite-design-system/components/tabs/

Reply:
awesome thanks for entertaining that for me